readme.md

```diff
@@ -278,13 +315,13 @@ exit
 ## Running the operator locally
```

Comment:
Be sure to add the following note

> Note: this operator build process is tested with [podman](https://podman.io/), but some of the build files (Makefile specifically) use docker because they are generated automatically by operator-sdk. It is recommended [remap the docker command to the podman command](https://developers.redhat.com/blog/2020/11/19/transitioning-from-docker-to-podman#transition_to_the_podman_cli).